.goreleaser.yml

```
@@ -30,6 +30,14 @@ before:
   hooks:
     - go mod tidy
 
+brews:
+  - repository:
+      owner: bomctl
+      name: homebrew-bomctl
```

Follow the steps [here](https://github.com/bomctl/bomctl/issues/35#issuecomment-2127725236), then add `branch:` and `token:`

```suggestion
  - repository:
      owner: bomctl
      name: homebrew-bomctl
      branch: "{{ .ProjectName }}-{{ .Tag }}"
      token: "{{ .Env.HOMEBREW_TAP_GITHUB_TOKEN }}"
```

---

It should only run without `--snapshot` when a tag is pushed, any normal push or pull_request event should run in snapshot mode

---

Have you tried it?

---

Could add something like this so that new brew releases create a PR instead of just pushing:

```yaml
      pull_request:
        enabled: true
        base:
          owner: bomctl
          name: homebrew-bomctl
          branch: main
```

---

Wouldn't we want goreleaser to only run in the core repo? I imagine it would fail if it ran in a fork, right?

---

I would hope so